python/tests/test_util/test_config_util.py

```diff
@@ -1,7 +1,7 @@
 from aiconfig.util.config_utils import get_api_key_from_environment
 
 
-def test_get_api_key_from_environment():
+def test_maybe_get_api_key_from_environment():
```

Comment:
```suggestion
def test_get_api_key_from_environment():
```

Reply:
I just deleted the entire file because it wasn't being used